src/frontend/admin/src/tests/product/product.test.e2e.ts

```diff
@@ -15,6 +15,10 @@ import { Course, DTOCourse } from "@/services/api/models/Course";
 import { mockCourseRunsFromCourse } from "@/tests/mocks/course-runs/course-runs-mocks";
 import { CourseRun } from "@/services/api/models/CourseRun";
 import { COURSE_OPTIONS_REQUEST_RESULT } from "@/tests/mocks/courses/course-mocks";
+import {
+  ContractDefinition,
+  DTOContractDefinition,
+} from "@/services/api/models/ContractDefinition";
 
 test.describe("Product list", () => {
```

Comment:
IMO it could be more complete to also add a test where the contract definition is created via the "+" button of the search input.